config/config.yaml

```diff
@@ -1,2 +1,4 @@
-project_code: aws-nextflow-workflows
-region: us-east-1
+project_code: aws-workflows-nextflow-infra
+profile: {{ var.profile | default("default") }}
+region: {{ var.region | default("us-east-1") }}
+aws_infra_templates_root_url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.3/templates
```

Comment:
I suggest the root be `https://raw.githubusercontent.com/Sage-Bionetworks` because there are things outside of aws-infra you might be interested, like lambdas published from Sage-Bionetworks-IT org.  Also you may want to reference different versions of  the CFN templates in each one of your sceptre templates.

Reply:
ohh wait, sorry.  ignore this comment.  i got mixed up with github reference and S3 bucket reference.  this is ok.

Reply:
Got it!

Reply:
sorry, the version thing i said in the first comment if valid, so i think the root should be.. `https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra`